src/env.py

```diff
@@ -154,6 +154,7 @@ def __list_parser(var: Optional[str]) -> list[str]:
 API_ID: Final = int(os.environ['API_ID']) if os.environ.get('API_ID') else None
 API_HASH: Final = os.environ.get('API_HASH')
 TOKEN: Final = os.environ.get('TOKEN')
+TELEGRAPH_IMG_UPLOAD: Final = os.environ['TELEGRAPH_IMG_UPLOAD'] if os.environ.get('TELEGRAPH_IMG_UPLOAD') else False
```

Comment:
Document it in `docs/advanced-settings.md`, `docker-compose.yml.sample` and `.env.smaple`.
And use `__bool_parser()` above.